monkestation/code/modules/virology/disease/plague_rat/plague_rat.dm

```diff
@@ -6,9 +6,17 @@
 	icon_living = "mouse_plague"
 	icon_dead = "mouse_plague_dead"
 
+	maxHealth = 300
```

Comment:
This seems rather excessive, especially considering this is on par with ice whelps, polar bears, and xeno defenders. Keep in mind plague rats can just go under players, go into vents, go through grilles, tables, and doors, and are small, making them difficult to hit to begin with.

Reply:
there's a proper counter to them: cats (including Runtime) will instantly splat any rats next to them, *including plague rats*

drag a cat around instead of just magdumping